### DISCUSSION TITLE
Google model viewer ar modes issue on ios

### DISCUSSION THREAD

**Discussion by pnkj1002** [google/model-viewer#4526] [Q&A]
### Description
I'm using google-model-viewer for AR quick look. I'm sending only quick-look on ar-modes but this is also giving option to view model in object viewer. Is there any option to disable that. It would also be better if there is any option to hide these buttons from top or are there any event to know I'm switching between scene viewer and quick look option.

### Version
- model-viewer: v3.1.1

### Browser Affected
- [x] Chrome
- [x] Edge
- [x] Firefox
- [x] IE
- [x] Safari

### OS
- [x] iOS

### AR
- [x] QuickLook
- 
![CHROME](https://github.com/google/model-viewer/assets/106859150/c8ae6f13-98a6-4195-afd2-90dad43dd015)

**Comment by elalish** [comment]
No, that's Apple's UI which we have no control over (it's not in the browser at all). There is no switching between Scene Viewer and Quick Look as the first is an Android app and the second is an iOS app.
